src/content/docs/style-guide/writing-docs/processes-procedures/create-preview-document.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Create content for the preview docs site
+contentType: page
+template: basicDoc
+topics:
+  - Basic style guide
+  - Writing guidelines
+redirects:
+---
+
+The preview docs site hosts preview docs for upcoming features that we share with customers.
+
+Unlike the public docs site, the preview docs site is not publicly editable and does not appear in search engine results. Additionally, the site is not indexed, so you can't navigate from one preview doc to another. Because the preview docs site is located in a different repo than the public docs site, contributing looks a little different.
+
+Before you get started, make sure you have done the following:
+
+## Before you write [#do-first]
+
+- Verify that you have access to the [beta-docs-site repo](https://github.com/newrelic/beta-docs-site). If you do not  have access, contact the #doc-eng-help Slack channel for help.
+- Clone the beta-docs-site repo on your computer.
+- Make sure you're working in the correct repo on Github Desktop.
+- Create a branch off the beta-docs-site repo to store your work.
+  
+## Create a new preview doc [#create-docs]
+
+Once in the beta-docs-site repo:
+
+1. In `/src/content/docs/`, find a good location for your doc.
+2. Using your text editor, create a new `.mdx` file or copy an existing doc.
+3. Write your content, making sure to include any info about accessing the preview feature you're documenting.
+4. Commit your changes and create a pull request.
+5. Share the preview link with reviewers and implement any feedback.
+6. Merge the final PR.
+7. Create the official link by pasting the path name after https://docs-preview.newrelic.com/docs/.(Example: https://docs-preview.newrelic.com/docs/new-relic-distribution-of-opentelemetry)
```

Comment:
```suggestion
7. Create the official link by pasting the path name after `https://docs-preview.newrelic.com/docs/` (For example: `https://docs-preview.newrelic.com/docs/new-feature-name`)
```

Let's replace the real OTEL preview link with a fake one (so the public won't have access to that doc).

Reply:
That makes sense. Wasn't sure about the formatting there or including the OTEL...